backend/spec/controllers/spree/admin/users_controller_spec.rb

```diff
@@ -59,6 +59,30 @@
         expect(assigns(:collection)).to eq [user]
       end
     end
+
+    context "when Spree.user_class have a different namespace than Spree" do
+      class UserModel < ApplicationRecord
+        self.table_name = 'spree_users'
+        include Spree::UserMethods
+      end
+
+      before do
+        Spree.user_class = 'UserModel'
```

Comment:
I think we should do an after block here too to make sure subsequent tests are using the original User.